pkg/core/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -319,6 +319,24 @@ one = 'Kafka instance "{{.Name}}" already exists'
 [kafka.create.error.oneinstance]
 one = 'unable to create new Kafka instance. Only one instance is allowed with an eval account'
 
+[kafka.create.region.error.regionNotSupported]
+one = '''Selected region does not support the instance types that you can create. Region {{.Region}}
+Your instance types {{.MyTypes}}. 
+Region instance types {{.CloudTypes}}  
```

Comment:
An output like this will read better (more naturally): 

```shell
the selected region "aws" does not support the instance types that you can create (eval, standard). Supported types for this region are: premium, trial
```

Reply:
Also error messages should begin with a lowercase.